### DISCUSSION TITLE
should there be a discussion category for development discussions?

### DISCUSSION THREAD

**Discussion by julie777** [pyscaffold/pyscaffold#596] [Dev Discussions]
I was about to start a new discussion and when I considered the categories I wondered if we should have one for development discussions.  This would include discussing how some part of the code works, or how we should implement something new, etc. It doesn't seem like something users would be that interested in. Currently I am going to put it in general which feels wrong somehow.

**Comment by FlorianWilhelm** [comment]
Thanks @julie777 for this suggestion. Sounds good to me, @abravalheri what do you think?

**Comment by abravalheri** [comment]
I think this approach works! 😄

**Comment by abravalheri** [comment]
Done! https://github.com/pyscaffold/pyscaffold/discussions/categories/dev-discussions 🥳 

Please feel free to bikeshed the name. I am always bad at naming.

**Reply by julie777** [reply]
I like the name.  I think you can close the discussion
